recipes/valgrind-macos/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+import sys
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+    generators = "VirtualRunEnv"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.tool_requires(self.tested_reference_str)
+
+    def test(self):
+        if can_run(self):
+            self.run("valgrind --version",
```

Comment:
Is this enough to confirm that valgrind works? It would be good to see output over a simple test executable here - in my experience `--vesion` can provide misleading results

Reply:
Yes you're right. I turned this PR into a draft because it doesn't even compile on macos 13